docs/available_commands.md

```diff
@@ -24,6 +24,7 @@ Currently, the command adds or modifies the following pieces of information:
 * Generates a *bom-ref* for components which don't have one, yet. The *bom-ref* will be a GUID.
 * If the path to a folder with license text files is provided, the text will be included in the SBOM, if the license has the corresponding `name`.
 * If a `license.name` is similar to an SPDX-ID, it will be replaced, e.g. `{"license": {"name": "The Apache License, Version 2.0"}}` leads to `{"license": {"id": "Apache-2.0"}}`. For this purpose a [JSON-file](https://github.com/Festo-se/cyclonedx-editor-validator/blob/main/cdxev/amend/license_name_spdx_id_map.json) is used, where we provide a mapping of license names to SPDX-IDs, based on this [license-mapping](https://github.com/CycloneDX/cyclonedx-core-java/blob/master/src/main/resources/license-mapping.json).
+* If neither `licenses` nor `copyright` exist but a supplier.name is present, the tool will create a copyright with the schema 'Copyright current year supplier.name, all rights reserved'. For example ´Copyright 2024 Acme. Inc., all rights reserved´ is created from the `{"supplier":{"name": "Acme Inc."}}` in the year 2024.
```

Comment:
Copyright string should start/end with quotation marks

Reply:
Done

Reply:
still missing at the general schema